pkg/db/postgres_controller.go

```diff
@@ -33,8 +38,13 @@ func (p *Postgres) DeleteNetwork(networkID uint) (int64, error) {
 
 	return result.RowsAffected, result.Error
 }
-func (p *Postgres) CreateComponent() {
 
+func (p *Postgres) CreateComponent(component *model.Component) (*model.Component, error) {
+	result := p.gDB.Create(component)
```

Comment:
There is a case that the code creates duplicated components.

Reply:
Could you explain what case it is?

Reply:
I am sorry. Let's me provide more detail idea. Basically, each networkid should have only one l2geth or ... So I think it is better if we check there is any l2geth for the networkid

Reply:
In our case, it is possible that there are 2 l2geth with the same networkid

Reply:
I understand! Thank you for the explanation. 👍

Reply:
Oh, I'm sorry.
It is my fault. My DB modeling is wrong. It had to prevent same networkid in case of same kind of component type.
Let's keep it in the code level. Let me add unique key about (network_id, type)

Thank you very much! 🙇🏼‍♂️